jina/flow/base.py

```diff
@@ -1558,6 +1558,28 @@ def rolling_update(self, pod_name: str, dump_path: Optional[str] = None):
                 f'The BasePod {pod_name} is not a CompoundPod and does not support updating'
             )
 
+    def dump(
+        self,
+        pod_name: str,
+        dump_path: str,
+        shards: int,
+        on_done=None,
+    ):
+        """
+        Send a dump request to the pod specified with pod_name
+
+        :param pod_name: pod to target with the dump request
+        :param dump_path: the path to which to dump
+        :param shards: nr of shards to dump for
+        :param on_done: the function to be called after the dump request has completed
+        """
+        self.post(
+            on='/dump',
+            target_peapod=pod_name,
+            parameters={'dump_path': dump_path, 'shards': shards},
```

Comment:
I think there needs to be a way to specify more dynamic argument here as well. Imagine parameters like `dump_format`. I am good either way: add a `**kwargs` to the function definition and then passed to the `parameters` dict or a `parameters` argument which is altered with `dump_path` and `shards` afterwards.

Reply:
According to Han's comment above, we are planning to remove this part of the code in https://github.com/jina-ai/jina/pull/2734